lib/pacemaker/pcmk_sched_colocation.c

```diff
@@ -136,10 +136,14 @@ cmp_primary_priority(gconstpointer a, gconstpointer b)
  *
  * \param[in,out] list        List of constraints to add \p colocation to
  * \param[in]     colocation  Colocation constraint to add to \p list
+ *
+ * \note The list will be sorted using cmp_primary_priority().
```

Comment:
Has to already be pre-sorted if non-empty too, since `g_list_insert_sorted()` assumes that. This is fine though